appcelerator/titanium_mobile
# TIMOB-2851: Android: Opacity property and animation not working as expected.

http://jira.appcelerator.org/browse/TIMOB-2851

Steps to test:
1. Use the second snippet from Fred's comment on the Jira ticket
2. Launch the app (It should start with a black background with a "toggle opacity" button)
3. Click on toggle opacity (you should see an animation that slowly colors a box red)

Before this patch, the "toggle opacity" button was not working the first time you click it.
